### ISSUE TITLE
Bug: data disappeared from the team

### ISSUE THREAD

**Issue by mkalininait** [rails-girls-summer-of-code/rgsoc-teams#669]
One applicant reported the following problem:

*"I am applying for RGSoC 2017 and had created my team [team_name] and received confirmation from both my coaches as well. My teammate and I had started filling out our application too, saving drafts periodically.

However, we are suddenly finding ourselves unable to see any information listed for our team and our application draft is unavailable too. We were able to view and edit all of this information even until a few hours ago."*

----

I investigated this problem.  It looks like all team members were somehow removed from the team at one point.  Because of this, the students lost access to their application. However, the application data persists in the DB.

I added one of the students back to the team so that they could add the team members again and submit the application. The question is: how did it happen that the data disappeared from the team?

Here is what I got from the production DB, the Users table:

![joxi_screenshot_1487941847332](https://cloud.githubusercontent.com/assets/1745735/23304733/a5ba6386-fa9b-11e6-8a1a-f4423a4cb546.png)

Ask me for more data.

----
Basically, all users were removed from the team in one go, but the roles stayed there. 
I can see three possible scenarios:
 1. The users have been removed by someone in the team
